packages/notification-center/src/components/popover-notification-center/PopoverNotificationCenter.tsx

```diff
@@ -21,6 +22,9 @@ interface IPopoverNotificationCenterProps {
   actionsResultBlock?: (templateIdentifier: string, messageAction: IMessageAction) => JSX.Element;
   tabs?: ITab[];
   showUserPreferences?: boolean;
+  offset?: number;
+  position?: PopoverProps['position'];
+  placement?: PopoverProps['placement'];
```

Comment:
`placement` prop is deprecated in Mantine's v5.0, will have to remove it in the future version
or maybe do we internally handle it and not expose the placement prop?

Reply:
Could you eloborate on what this prop actually does? And if Mantine replaces it, they replace it with something else than?

Reply:
They have merged the `position` and `placement` props to just the `position` prop
`placement` prop used to accept `start`, `center` & `end`
now `position` prop along with old `top`, `bottom` etc values also accepts `top-start`, `top-end` and so on in the new version.

Reply:
So maybe we can remove it too? And map the functionality behind the scenes so the user is not aware of it, and when we will update Mantine, we can just remove it. Wdyt?

Reply:
Yeah, was thinking the same, will map this prop with the future version then.